src/jobs/config_sync_job.ts

```diff
@@ -377,6 +489,133 @@ export class ConfigSyncJob extends SyncJob {
     // if any of those operations did fail, return false
     return operationsOk;
   }
+
+  /*
+  checkes if we sync timer-tiket or ticket-ticket tools
+  returns true if ticket-ticket, false othewise
+  */
+  private _isTicket2TicketSync() {
+    return (this._connection.firstService.name === 'Jira' || this._connection.firstService.name === 'Redmine')
+      ? (this._connection.secondService.name === 'Jira' || this._connection.secondService.name === 'Redmine')
+      : false
+  }
```

Comment:
Tohle by asi mělo být v nějaké buď base třídě nebo utility třídě. Taky úplně nevidím dobrou rozšiřitelnost tohoto řešení. Já bych to asi napsal jako return `ticketSystems.includes(this_connection.firstService.name) &&  ticketSystems.includes(this_connection.secondService.name) `

Reply:
Ono to v jako utility funkce už existovalo v src/shared/ticket2ticket_service.ts, tak6e jsem to odstranil a místo toho includnul tu sdílenou